flac_codec_registration.src.html

```diff
@@ -94,6 +102,59 @@
 NOTE: Once the initialization has succeeded, any FLAC packet can be decoded at
     any time without error, but this might not result in the expected audio output.
 
+AudioEncoderConfig extensions {#audioencoderconfig-extensions}
+=============================================================
+
+<pre class='idl'>
+<xmp>
+partial dictionary AudioEncoderConfig {
+  FlacEncoderConfig flac;
+};
+</xmp>
+</pre>
+
+<dl>
+  <dt><dfn dict-member for=AudioEncoderConfig>flac</dfn></dt>
+  <dd>
+    Contains codec specific configuration options for the FLAC codec.
+  </dd>
+</dl>
+
+
+FlacEncoderConfig {#flac-encoder-config}
+--------------------------------------
+
+<pre class='idl'>
+<xmp>
+dictionary FlacEncoderConfig {
+  [EnforceRange] unsigned long blockSize = 0;
+  [EnforceRange] unsigned long compressLevel = 5;
+};
+</xmp>
+</pre>
+
+To check if an {{FlacEncoderConfig}} is valid, run these steps:
+1. If {{FlacEncoderConfig/blockSize}} is not a valid block size,
+	which is described section 5.1 of [[FLAC-DRAFT]], return `false`.
```

Comment:
Do we need both the `FLAC` and `FLAC-DRAFT` references? The FLAC reference also describes the valid block size. That said, if this is going through the IETF, we should prefer the IETF-DRAFT, then update the registration once it becomes an RFC

Reply:
IETF-DRAFT should be more formal, we will record this, and update when it become final RFC.